src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -127,6 +126,7 @@ object EasyIngest {
     for {
       foxml <- getFOXML(sdo)
       pid <- executeIngest(configDictionary(sdo.getName), foxml)
+      _ = log.info(s"ingested $pid $foxml")
```

Comment:
I guess fail fast and roll-back would be appropriate here.
With roll-back I assume you mean deleting already ingested content of a failing deposit?